airbyte-json-validation/src/main/java/io/airbyte/validation/json/JsonSchemaValidator.java

```diff
@@ -44,6 +46,7 @@ public class JsonSchemaValidator {
 
   private final JsonSchemaFactory jsonSchemaFactory;
   private final URI baseUri;
+  private final Map<String, JsonSchema> schemaToValidators = new HashMap<>();
```

Comment:
Potential here for a threading issue, but I don't believe we currently utilize this class in multiple threads currently.

Reply:
that's right. I'm thinking of leaving as is for now. how is that?